gedcom/people.py

```diff
@@ -155,6 +155,29 @@ def _is_valid_age(self, person):
             else:
                 return True
 
+    def _is_valid_date_birthdeath(self, person):
+        """ checks if birth date is before current date
+        """
+        if self._curr_person["birth_date"] is not None:
```

Comment:
Since you are passing in the person object, you should probably be using that object instead of `self._curr_person`